kalite/distributed/management/commands/restorebackup.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from django.core.management.base import BaseCommand
+from django.core.management import call_command
+from django.conf import settings
+
+from optparse import make_option
+
+
+class Command(BaseCommand):
+    help = "Restore backup to a previous state."
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--file',
+            dest='backup_filepath',
+            default="",
+            help="File path"),
+        )
+    
+    def handle(self, *args, **options):
+            if(options['backup_filepath']):
+                if(os.path.isfile(backup_filepath)):
```

Comment:
Does this work? `backup_filepath` doesn't seem to be assigned anywhere within this file.